### PR TITLE
chore: update with reformatted config

### PR DESCRIPTION
**Pull request by chloeminkyung**

### CODE REVIEW

File: examples/1_hypercharge_postgres/dozer-config.yaml [hunk 1]
```diff
@@ -41,14 +61,13 @@ sources:
 endpoints:
   - name: stocks
     path: /stocks
-    table_name: stocks
```
Comment (v3g42) [inline examples/1_hypercharge_postgres/dozer-config.yaml:44]: why dont you let one API connect directly to source but select the columns at the source level and another transformed using SQL. 

And put comments near the endpoints explaining how they work ?

File: examples/2_eth_stats_sample/dozer-config.yaml [hunk 2]
```diff
@@ -23,13 +23,26 @@ connections:
     name: eth_conn
 
 sql: |
-  select punkIndex, minValue, toAddress
-  INTO punk_offered from punks_PunkOffered;
+  SELECT
```
Comment (v3g42) [inline examples/2_eth_stats_sample/dozer-config.yaml:26]: I think we should add comments for each of these too. So users can understand when they see it.

File: examples/1_hypercharge_postgres/dozer-config.yaml (hunk 1 above)
Comment (chloeminkyung) [reply examples/1_hypercharge_postgres/dozer-config.yaml:44]: sure - will do

File: examples/2_eth_stats_sample/dozer-config.yaml (hunk 2 above)
Comment (chloeminkyung) [reply examples/2_eth_stats_sample/dozer-config.yaml]: ok

File: examples/1_hypercharge_postgres/dozer-config.yaml [hunk 3]
```diff
@@ -38,17 +38,31 @@ sources:
...
     index:
       primary_key:
         - id
-  - id: null
-    name: stocks_meta
+  - name: stocks_meta
     path: /stocks-meta
     table_name: stocks_meta
+    # direct selecting columns at the source level for `stocks_meta`
+    sql: SELECT * from stocks_meta;
```
Comment (v3g42) [inline examples/1_hypercharge_postgres/dozer-config.yaml:65]: I m guessing this will fail ? There is no SQL property on endpoins anymore.

Comment (chloeminkyung) [reply examples/1_hypercharge_postgres/dozer-config.yaml]: addressed

**Review by v3g42 — APPROVED** [review]

**Pull request merged**